File: lua/ht/plugins/external/octo-nvim.lua
return {
  {
    "pwntester/octo.nvim",
    lazy = true,
    dependencies = {
      "nvim-lua/plenary.nvim",
      "TwIStOy/telescope.nvim",
      "nvim-tree/nvim-web-devicons",
    },
    cmd = { "Octo" },
    config = true,
    opts = {
      default_remote = { "origin", "upstream" },
    },
  },
}

File: src/conf/plugins/other/octo-nvim.ts
import { Plugin } from "@core/model";

export const plugin = new Plugin({
  shortUrl: "pwntester/octo.nvim",
  lazy: {
    dependencies: [
      "nvim-lua/plenary.nvim",
      "nvim-telescope/telescope.nvim",
      "nvim-tree/nvim-web-devicons",
    ],
    cmd: ["Octo"],
    opts: {
      default_remote: ["origin", "upstream"],
    },
    config: true,
  },
});
